packages/@react-spectrum/meter/docs/Meter.mdx

```diff
@@ -0,0 +1,110 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/meter';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Meter} from '@react-spectrum/meter';
+```
+
+# Meter
+
+<p>{docs.exports.Meter.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Meter']}
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/meter/'}]} />
+
+## Example
+
+```tsx example
+<Meter label="Storage space" variant="positive" value={35} />
+```
+
+## Value
+
+By default Meters are controlled with the `value` prop representing current percentage (as an integer).
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<Meter label="Storage space" value={25} />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<Meter label="Widgets Used" minValue={50} maxValue={150} value={100} />
+```
+
+Values are formatted by default as a percentage, but can modified via `formatOptions` for different formats.
+The `formatOptions` are displayed via the [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) applied for the current locale.
+
+```tsx example
+<Meter label="Currency" formatOptions={{style: 'currency', currency: 'JPY'}} value={60} />
+```
+
+### Accessibility
+
+Meters are made accessible as per the ["meter" role in WAI-ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/#meter) and will fallback to the ["progressbar" role](https://www.w3.org/TR/wai-aria-1.2/#progressbar) if the meter role is not supported.
+Depending on the visualization being used (ie depending on the `showValueLabel` prop), a `label`, `aria-label`, or `aria-labelledby` attribute are required.
+
+```tsx example
+<div>
+  <Meter aria-labelledby="pc-label" showValueLabel={false} />
+  <label id="pc-label" style={{display: 'block'}}>Meter</label>
+</div>
+```
+
+### Internationalization
+
+To internationalize a Meter, a localized string should be passed to the `label` prop, `aria-label` prop, or value of associated `aria-labelledby` element.
+
+For RTL (right-to-left) languages, the layout of the bar loader is mirrored.
+The placement of the percent sign differs depending on the locale.
+
+If the prop `formatOptions` is used, Meter will utilize current locale to format the value string.
+
+## Props
+
+<PropTable component={docs.exports.Meter} links={docs.links} />
+
+## Visual Options
+
+### Labels
+
+Value labels are shown by default on top, but they can also be moved to the side or hidden as needed.
```

Comment:
Maybe valuable to include either https://spectrum.adobe.com/page/meter/#Labels since it mentions that the label should always be sentence case etc